### ISSUE TITLE
Heap summary viewer crashes with detailed info on

### ISSUE THREAD

**Issue by GacekKosmatek** [lucko/spark-viewer#40]
Having the local storage `prefShowInfo` value set to `true` makes the heap summary viewer crash and show a blank page

Using spark included with Purpur build 1691 (probably latest version of spark?)

Stacktrace:
```
TypeError: Cannot read properties of undefined (reading 'numberOfIncludedTicks')
    at z (meta.js:84:59)
    at oo (react-dom.production.min.js:157:137)
    at Zu (react-dom.production.min.js:267:460)
    at Li (react-dom.production.min.js:250:347)
    at Ni (react-dom.production.min.js:250:278)
    at Pi (react-dom.production.min.js:250:138)
    at bi (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Ql (react-dom.production.min.js:122:325)
```

**Comment by GacekKosmatek** [comment]
After trying to debug the issue in a local environment it looks like the metadata doesn't have the `dataAggregator` value (file `src/viewer/meta.js` line 84)

**Comment by GacekKosmatek** [comment]
Submitted a PR that fixes this issue (#41)
